R/zchunk_L102.GDP.R

```diff
@@ -87,9 +87,9 @@ join.gdp.ts <- function(past, future, grouping) {
 #' by GCAM region.
 #'
 #' The scenarios generated include the SSPs, the gSSPs (SSPs modified by
-#' near-term IMF projections), and, for some reason, something derived from
-#' GCAM3.  GDP outputs are in millions of 1990 USD, which presumably means they
-#' are MER values.  Per-capita values are in thousands of 1990 USD.
```

Comment:
Sorry about this.  These were part of notes to myself about things to follow up on.  I cleaned out most of them, but missed these.